crypto/batchverifier.go

```diff
@@ -120,18 +120,18 @@ func (b *BatchVerifier) Verify() error {
 // if some signatures are invalid, true will be set in failed at the corresponding indexes, and
 // ErrBatchVerificationFailed for err
 func (b *BatchVerifier) VerifyWithFeedback() (failed []bool, err error) {
-	if b.getNumberOfEnqueuedSignatures() == 0 {
+	if b.GetNumberOfEnqueuedSignatures() == 0 {
 		return nil, nil
 	}
-	var messages = make([][]byte, b.getNumberOfEnqueuedSignatures())
+	var messages = make([][]byte, b.GetNumberOfEnqueuedSignatures())
 	for i, m := range b.messages {
 		messages[i] = HashRep(m)
```

Comment:
Not your code, but I'm starting to pay more attention to this. Since the things in `b.messages` are Transactions, mostly.  This performs a 1300 byte copy for each member of the for loop, to fill `m`. Then it performs _another_ 1300 byte copy to call `HashRep`.

I think I'd advocate fixing that first copy here.
```suggestion
	for i := range b.messages {
		messages[i] = HashRep(b.messages[i])
```